install.sh

```diff
@@ -200,6 +204,38 @@ file_not_grpowned() {
   [[ " $(id -G "$USER") " != *" $(get_group "$1") "*  ]]
 }
 
+# Please sync with 'test-ruby()' in 'Library/Homebrew/utils/ruby.sh' from Homebrew/brew repository.
+test-ruby () {
+  [[ ! -x $1 ]] && { echo "false"; return 1; }
+  "$1" --enable-frozen-string-literal --disable=gems,did_you_mean,rubyopt -rrubygems -e \
+    "puts Gem::Version.new(RUBY_VERSION.to_s.dup).to_s.split('.').first(2) == \
+          Gem::Version.new('$required_ruby_version').to_s.split('.').first(2)" 2>/dev/null
```

Comment:
Capture the output and compare its output to the word `true` and return `0` if equal and `1` otherwise.

Reply:
Same as above -- I actually implemented what you suggested here but decided that keeping it identical to its sibling in Homebrew/brew is more important.

Reply:
I'm 👍  for this change if we also change it in `ruby.sh` in Homebrew/brew.

Reply:
Let's change both!